src/sourmash/search.py

```diff
@@ -11,6 +11,25 @@
 from .sketchcomparison import FracMinHashComparison, NumMinHashComparison
 
 
+def calc_threshold_from_bp(threshold_bp, scaled, query_size):
```

Comment:
Is it possible for the `threshold_bp` to be larger than the `query_size`? Throwing an exception would be helpful here.

Reply:
yes, good catch! this is addressed over in #2116, where I refactor some other code to use this same function:

https://github.com/sourmash-bio/sourmash/pull/2116/files#diff-51443303a0e6c38023d8977d891df4ae18dc06a2431d29338fa8648a042132bfR33